data/101/749/159/101749159.geojson

```diff
@@ -839,7 +839,7 @@
     "wof:lang":[
         "dan"
     ],
-    "wof:lastmodified":1587428785,
+    "wof:lastmodified":1607124774,
     "wof:megacity":1,
     "wof:name":"Copenhagen",
```

Comment:
Should this be the capital of the Capital Region?

Reply:
Now, Hillerød is the capital of the Capital Region... so why was this feature touched? Should we say it's the country capital?

Reply:
A bit confusing, but [Hillerød](https://en.wikipedia.org/wiki/Hiller%C3%B8d) is actually the capital of the "Capital Region" in Denmark, even though Copenhagen is the capital of Denmark. Regardless, this feature should not have been updated in this commit (but will be in a future commit, see comment about lists v strings below).